tests/test-bundle-connects.stanza

```diff
@@ -0,0 +1,50 @@
+#use-added-syntax(jitx, tests)
+defpackage ocdb/tests/bundle-connects: 
+  import core
+  import collections
+  import jitx
+  import jitx/commands
+  import ocdb/connects
+  import ocdb/bundles
+
+  import ocdb/defaults
+  import ocdb/design-vars
+  import ocdb/generator-utils
+  import ocdb/generic-components
+  import ocdb/property-structs
+  import ocdb/checks
+  import ocdb/micro-controllers
+    
+deftest connect-spi-01 :
+  pcb-module accel:  
+    ; connect mcu spi with cip and copi to accelerometer spi using SDI and SDO
+    inst accel : ocdb/st-microelectronics/LIS3DH/module
+    inst mcu : ocdb/st-microelectronics/STM32F105VBH6/module([])
+    require spi1:spi-controller() from mcu.mcu
+    connect-spi(mcu.mcu, spi1, [accel.spi])
+
+    require spi2:spi-controller() from mcu.mcu
+    inst spi-other-mcu : ocdb/st-microelectronics/STM32F105VBH6/module([])
+    require spi-from-other:spi-controller() from spi-other-mcu.mcu
+    inst adc : ocdb/analog-devices/AD7124-8BBCPZ/module[4]
+
+    ; connet main mcu to other mcu with spi-controller and other peripheral spi
+    connect-spi(mcu.mcu, spi2, [spi-from-other adc[0].spi adc[1].spi adc[2].spi adc[3].spi])
+    ; check spi cipo sdo, copi sdi, sck, and gpio for cs
+    ; spi 1 checks
+    #ASSERT(connected?([accel.spi.sdi spi1.copi]))
```

Comment:
`#EXPECT`. The difference between `#EXPECT` and `#ASSERT` is that `#EXPECT` will not stop the test even if the condition is false